app/templates/components/preprint-form-header.hbs

```diff
@@ -96,9 +98,36 @@
                 </ul>
             {{else if (eq name 'COI')}}
                 <p>
-                    {{if hasCoi (t "submit.body.conflict_of_interest.yes") (t "submit.body.conflict_of_interest.no")}}: 
+                    <em class="m-r-md">{{if hasCoi (t "submit.body.conflict_of_interest.yes") (t "submit.body.conflict_of_interest.no")}}:</em>
                     {{if (and hasCoi coiStatement) coiStatement (t "submit.body.conflict_of_interest.placeholder")}}
                 </p>
+            {{else if (eq name 'Assertions')}}
+                {{#if sloanDataInputEnabled}}
+                    <p><em class="m-r-md">{{t "components.author-assertions.public_data.title"}}:</em> 
+                        {{#if (eq hasDataLinks "available")}}
+                            {{t "components.author-assertions.available.available"}}
+                        {{/if}}
+                        {{#if (eq hasDataLinks "no")}}
+                            {{t "components.author-assertions.available.no"}}
+                        {{/if}}
+                        {{#if (eq hasDataLinks "not_applicable")}}
+                            {{t "components.author-assertions.available.not_applicable"}}
+                        {{/if}}
+                    </p>
```

Comment:
According to the mockups (saved states), you should add another `<p>` here that holds the actual value (based) on the model? So if it's `available`, you should have the links listed here; if it's `no`, you should have the `whyNo*` or the placeholder, and so forth.